test/api/server-check.test.js

```diff
@@ -130,3 +130,10 @@ describe("Env util package.json check", () => {
     ).toEqual(true);
   });
 });
+
+// ["/dashboard","/printers","/filemanager","/history","/mon/panel","/mon/camera","/mon/printerMap","/mon/list","/mon/currentOp","/filament","/system","/serverChecks/amialive","/printers/add","/printers/update","/printers/remove","/printers/removefile","/printers/removefolder","/printers/resyncFile","/prin
```

Comment:
Added the routes, thought nice place to put here in server tests, normally I would add swagger.json but I couldnt be bothered